### ISSUE TITLE
Strange Exception error on sample

### ISSUE THREAD

**Issue by mmauri** [Giorgi/DuckDB.NET#12]
Hi, 
The idea to have access from c# to DuckDB performance & its easiness to read parquet files looks amazing, but I tried to run this sample from the readme, and it returns a strange exception with the following message:
**"plú=:☻"**

this is the code producing it:
(using windows amd64 duckdb.dll v0.2.8)
```C#
 using (var conn = new DuckDBConnection("Data Source=:memory:"))
  {
      try
      {
          conn.Open();
          var command = conn.CreateCommand();
          command.CommandText = "CREATE TABLE integers(foo INTEGER, bar INTEGER);";
          var executeNonQuery = command.ExecuteNonQuery();
      }
      catch (Exception ex)
      {
          Console.WriteLine(ex.Message);
      }
  }
```

Thanks,

**Comment by Giorgi** [comment]
The [duckdb_result](https://github.com/duckdb/duckdb/blob/master/src/include/duckdb.h#L138) struct has changed so that's why the result is corrupted.

I will update the library.

**Comment by Giorgi** [comment]
Fixed in 070aade3593302afdd7e16283606833373d14907

**Comment by Giorgi** [comment]
@mmauri New version of NuGet packages uploaded. They will become available in couple of minutes.
